src/uniprotkb/components/entry/Entry.tsx

```diff
@@ -91,19 +93,31 @@ const Entry: FC = () => {
     [data]
   );
 
-  const sections = useMemo(
-    () =>
+  const sections = useMemo(() => {
+    return (
       transformedData &&
-      UniProtKBEntryConfig.map((section) => ({
-        label: section.name,
-        id: section.id,
-        disabled:
-          section.name === EntrySection.ExternalLinks
-            ? !hasExternalLinks(transformedData)
-            : !hasContent(transformedData[section.name]),
-      })),
-    [transformedData]
-  );
+      UniProtKBEntryConfig.map((section) => {
+        const taxId =
```

Comment:
I would just move the definition of taxId and numberOfIsoforms outside of the map (but inside the useMemo), as this will be the same value at every loop of the map